lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -29,6 +29,7 @@
     "disableStorageReset": false,
     "emulatedFormFactor": "mobile",
     "channel": "cli",
+    "budgetsPath": null,
```

Comment:
I think we want to update this to `budgetsJSON` as mentioned in https://github.com/GoogleChrome/lighthouse/pull/8427/files#r277141148

and maybe we could throw a real budget in here to flex its muscles 💪